inventory/service/group_vars/k8s-controller.yaml

```diff
@@ -291,7 +291,7 @@ helm_chart_instances:
     repo_name: opensearch
     name: opensearch
     ref: opensearch/opensearch
-    version: 1.17.0
+    version: 2.6.0
```

Comment:
Do you understand that here is the helm chart version (chart v2.6.0 => opensearch v2.3.0)? I do not get if we update the chart why not to pick the latest